tests/apm/overview.phpt

```diff
@@ -96,9 +102,9 @@ object(MongoDB\Driver\Monitoring\CommandFailedEvent)#%d (%d) {
     ["ok"]=>
     float(0)
     ["errmsg"]=>
-    string(12) "ns not found"
+    string(13) "can't find ns"
     ["code"]=>
-    int(26)%A
+    int(8)%A
```

Comment:
The fact that this still reports error code 8 ("UnknownError") instead of 26 ("NamespaceNotFound") is more evidence that `geoNear` is not being actively maintained and we should probably avoid using it for any testing.

c.f. https://github.com/mongodb/mongo/blob/master/src/mongo/base/error_codes.err